src/Microsoft.Health.Dicom.Web/Microsoft.Health.Dicom.Web.csproj

```diff
@@ -32,4 +32,10 @@
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
+  <Target Name="SwaggerPostBuildTarget" AfterTargets="Build" Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <Exec Command="dotnet tool restore"></Exec>
+    <Exec Command="dotnet swagger tofile  --yaml --output ..\..\swagger\v1\swagger.yaml bin\x64\Debug\net6.0\Microsoft.Health.Dicom.Web.dll v1"></Exec>
```

Comment:
Is there a way to unify this and the CI script? Could we instead always generate the files?